# Cloth.msg
std_msgs/Header header
uint32 rows
uint32 cols
float64[] Coll_Flag
float64[] Depth_Image
float64[] Depth_Cloth

